source/docs/reference/debugging.md

```diff
@@ -0,0 +1,93 @@
+---
+id: debugging
+title: Debugging
+description: Debug your Front-Commerce app.
+---
+
+# Debugging
+
+Debugging your code is an important part of the development process. This documentation explains how you can debug your Front-Commerce frontend and backend code using either the [VS Code debugger](https://code.visualstudio.com/docs/editor/debugging) or [Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools).
+
+Any debugger that can attach to Node.js can also be used to debug a Front-Commerce application. You can find more details in the Node.js [Debugging Guide](https://nodejs.org/en/docs/guides/debugging-getting-started/).
+
+## Debugging with VS Code
+
+Create a file named `.vscode/launch.json` at the root of your project with the following content:
+
+```json
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "FC: debug server-side",
+      "type": "node-terminal",
+      "request": "launch",
+      "command": "npm run start -- --inspect"
+    },
+    {
+      "name": "FC: debug client-side",
+      "type": "pwa-chrome",
+      "request": "launch",
+      "url": "http://localhost:4000"
+    },
+    {
+      "name": "FC: debug full stack",
+      "type": "node-terminal",
+      "request": "launch",
+      "command": "npm run start -- --inspect",
+      "console": "integratedTerminal",
+      "serverReadyAction": {
+        "pattern": "started server on .+, url: (https?://.+)",
+        "uriFormat": "%s",
+        "action": "debugWithChrome"
+      }
+    }
+  ]
+}
+```
+
+If you're [changing the port number](/docs/reference/environment-variables.html#Host) your application starts on, replace the `4000` in `http://localhost:4000` with the port you're using instead, you can use the url defined in your `FRONT_COMMERCE_URL`
+
+Now in VS Code go to the Debug panel (<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>D</kbd> on Windows/Linux, <kbd>⇧</kbd>+<kbd>⌘</kbd>+<kbd>D</kbd> on macOS), select a launch configuration, then press <kbd>F5</kbd> or select **Debug: Start Debugging** from the Command Palette to start your debugging session.
```

Comment:
For advanced settings or understand how it works in detail, I think it makes sense to link to the official documentation here (as it will be where developers will stop when reading the documentation).

```suggestion
Now in VS Code go to the Debug panel (<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>D</kbd> on Windows/Linux, <kbd>⇧</kbd>+<kbd>⌘</kbd>+<kbd>D</kbd> on macOS), select a launch configuration, then press <kbd>F5</kbd> or select **Debug: Start Debugging** from the Command Palette to start your debugging session. See [Debugging in Visual Studio Code (official documentation)](https://code.visualstudio.com/docs/editor/debugging) for more details.
```